packages/suite/src/views/dashboard/components/PortfolioCard/index.tsx

```diff
@@ -50,7 +50,7 @@ const PortfolioCard = React.memo(() => {
     if (discoveryStatus && discoveryStatus.status === 'exception') {
         body = <Exception exception={discoveryStatus} discovery={discovery} />;
     } else if (discoveryStatus && discoveryStatus.status === 'loading') {
-        body = <Loading />;
+        body = <SkeletonTransactionsGraph data-test="@dashboard/loading" />;
```

Comment:
animate prop might be missing

Reply:
it is fine here. it is optional. without it it should fallback to behaviour from useLoadingSkeletons that is already used in the `SkeletonTransactionsGraph `, there was just a bug where I don't actually use that value 😁 
Thank god you noticed.